code/game/objects/structures/crates_lockers/crates.dm

```diff
@@ -19,6 +19,14 @@
 	pass_flags_self = PASSSTRUCTURE | LETPASSTHROW
 	var/crate_climb_time = 20
 	var/obj/item/paper/fluff/jobs/cargo/manifest/manifest
+	/// Where the Icons for lids are located.
+	var/lid_icon = 'icons/obj/storage/crates.dmi'
+	/// Gives a crate a lid sprite if it has one. Leave Blank if it doesn't.
+	var/lid_icon_state
+	/// Controls the X value, allowing left and right pixel movement.
+	var/lid_x = 0
+	/// Controls the Y value, allowing up and down pixel movement.
```

Comment:
```suggestion
	/// Controls the X value of the lid, allowing left and right pixel movement.
	var/lid_x = 0
	/// Controls the Y value of the lid, allowing up and down pixel movement.
```
Probably goes without saying but just in case